utility/gmusic_playlists_to_plex.py

```diff
@@ -98,10 +109,11 @@ def main():
             playlistContent = []
             shareToken = pl['shareToken']
             # Go through tracks in Google Music Playlist
-            for ggmusic in mc.get_shared_playlist_contents(shareToken):
-                title = str(ggmusic['track']['title'].encode('ascii', 'ignore'))
-                album = str(ggmusic['track']['album'].encode('ascii', 'ignore'))
-                artist = str(ggmusic['track']['artist'])
+            for ggmusic in pl['tracks']:
+                info = get_songs_info(ggmusic['trackId'])
```

Comment:
`info` > `googleInfo` or something similar to denote that the variable is google related vs plex related.